melkor/models/sklearn_pipeline.py

```diff
@@ -0,0 +1,110 @@
+import sklearn
+import pandas as pd
+import numpy as np
+import pickle
+from nptyping import NDArray, Float32
+from typing import Union
+from sklearn.pipeline import Pipeline
+from sklearn.compose import ColumnTransformer
+
+# TODO: Add scoring metrics
+# TODO: Change y to 1d array
+
+
+class SKLearnModelPipeline:
```

Comment:
As a general comment, always document your classes, functions and modules, it is a bit difficult sometimes to reverse engineer the meaning and restrictions of the parameters